internal/resources/standalone/index-template.html

```diff
@@ -97,6 +97,16 @@
       padding: 24px;
     }
     </style>
+    <script>
+    // ZOMG this is awful. This beast comes from https://developer.mozilla.org/en-US/docs/Web/API/document/cookie
+    var cookieValue = document.cookie.replace(/(?:(?:^|.*;\s*)_grpcui_csrf_token\s*\=\s*([^;]*).*$)|^.*$/, "$1");
+    // add CSRF token to all non-GET requests
+    $(document).ajaxSend(function(event, request, settings) {
+        if (settings.method !== "GET") {
```

Comment:
Is there any reason to not include the CSRF token in GET requests? It seems like that wouldn't hurt, and it would provide a level of safety that protects side effect inducing GET requests that get created inadvertently. Also, the wrapper that validates the CSRF token seems to not skip GET requests.

Reply:
I guess not. I can have it apply to everything, it will just be ignored.

The thing that validates CSRF is a wrapper that only a protects a single handler, the `grpcui.RPCInvokeHandler`, which only accepts POST requests. (It results in "405 Method Not Allowed" for anything else.)

Reply:
FWIW, I formulated it this way so it only adds the header for requests where it will actually get validated. The only non-GET XHRs that this thing does is for invocations. The GET XHRs are for a different handler (one that does not check CSRF token because it isn't necessary -- it's just for querying RPC schema metadata down to the client).